sklearn/decomposition/tests/test_nmf.py

```diff
@@ -589,12 +707,66 @@ def test_nmf_custom_init_dtype_error():
     W = rng.random_sample((20, 15))
 
     with pytest.raises(TypeError, match="should have the same dtype as X"):
-        NMF(init='custom').fit(X, H=H, W=W)
+        Estimator(init='custom').fit(X, H=H, W=W)
 
     with pytest.raises(TypeError, match="should have the same dtype as X"):
         non_negative_factorization(X, H=H, update_H=False)
 
 
+def test_nmf_is_minibatch_nmf():
```

Comment:
I would not say "is" which has a specific meaning in python but rather "test_nmf_minibatchnmf_equivalence" or something like that

Reply:
Fixed.